### PR TITLE
added one finger zooming setting

### PR DESCRIPTION
**Pull request by gh123man**
This feature was very important to me and I was sad to see it disappear. 

Instead of outright removing it I thought it may be good to have a switch for those who prefer one finger zooming vs double tap zooming. 

I am not sure what your localization process is, please let me know how I can improve this change.

### CODE REVIEW

File: app/src/main/kotlin/com/simplemobiletools/gallery/helpers/Config.kt [hunk 1]
```diff
@@ -168,6 +168,10 @@ class Config(context: Context) : BaseConfig(context) {
         get() = prefs.getInt(getDirectoryColumnsField(), getDefaultDirectoryColumnCount())
         set(dirColumnCnt) = prefs.edit().putInt(getDirectoryColumnsField(), dirColumnCnt).apply()
 
+    var oneFingerZoom: Boolean
+        get() = prefs.getBoolean(ONE_FINGER_ZOOM, false)
+        set(loop) = prefs.edit().putBoolean(ONE_FINGER_ZOOM, loop).apply()
```
Comment (tibbi) [inline app/src/main/kotlin/com/simplemobiletools/gallery/helpers/Config.kt]: rename "loop" variables to "oneFingerZoom"

File: app/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -146,6 +146,7 @@
 
         This app is just one piece of a bigger series of apps. You can find the rest of them at http://www.simplemobiletools.com
     </string>
+    <string name="one_finger_zoom">Toggle one finger zoom</string>
```
Comment (tibbi) [inline app/src/main/res/values/strings.xml]: move this at the settings section, right under manage_extended_details. Add it to every language file, even if in english obviously. Ye, I have to keep adding it like that too :/

**Comment by tibbi** [comment]
added some comments, thanks

**Comment by gh123man** [comment]
Thanks for the feedback - Updates posted.

**Comment by tibbi** [comment]
cool, thanks. I would actually rename "toggle one finger zoom" to "Allow one finger zoom at fullscreen media", just to make it clearer. Sorry :x

**Comment by gh123man** [comment]
Not a problem :smile:

**Comment by tibbi** [comment]
alright, looks cool. I just moved it above the 2 Extended details setting buttons as they are related. Thanks!

**Pull request merged**